src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java

```diff
@@ -72,8 +72,8 @@ public int compareTo2(Object aObject) // empty line - ok
      * @param result
      * @return
      */
-    public static <T> Callable<T> callable(Runnable task, T result) // violation
-    {
+    public static <T> Callable<T> callable(Runnable task, T result)
+    { // violation above
```

Comment:
Please use trailing/same-line  comment.
If you do not have enough space, please shorten methods signature, as it can be any for this Check.

Same for other code like this.